pages/edit_group.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.page import PageRegion
+from pages.base import Base
+
+
+class Group(Base):
```

Comment:
If this is a page object we should follow the existing convention and name it `EditGroupPage`.